### PR TITLE
Basic database schema

### PR DESCRIPTION
**Pull request by vr2262**
- A `Video` has a `url_id` and `video_metadata`.
- A `TrackMatch` has a `track_id`, a `match_id`, and whether the track is `audio` or `video` (the matching track is the opposite). It also has their scores and seek times.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,6 @@
 numpy==1.11.1
 pafy==0.5.2
+psycopg2==2.6.2
+sqlalchemy==1.0.14
 tornado==4.4.1
 youtube_dl==2016.8.17
```
Comment (myf) [inline requirements.txt]: this (youtube-dl) will change at a very frequent rate, i think we should not pin it at 8.17

Comment (vr2262) [inline requirements.txt]: Actually... you're right. In this case I think it makes sense to allow it to be whatever.

**Comment by vr2262** [comment]
brb

**Comment by vr2262** [comment]
@myf I changed `Track` to `Video` so that you don't have to make 2 entries into the table for every video.

**Comment by vr2262** [comment]
@myf this one?

**Pull request merged**